packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -180,6 +184,23 @@ export const NewCommentNotification = registerNotificationType({
   },
 });
 
+// New comment on a subforum you're subscribed to.
+export const NewSubforumCommentNotification = registerNotificationType({
+  name: "newSubforumComment",
+  userSettingField: "notificationSubforumUnread",
+  allowedChannels: ["none", "email"],
+  async getMessage({documentType, documentId}: {documentType: string|null, documentId: string|null}) {
+    // NOTE: Currently we only allow notifications by email to avoid double notifying (with both the sidebar unread count
+    // and the notification), if we ever want to allow notifications in the sidebar we'll probably want to update this message
+    // and double check the links work
+    let document = await getDocument(documentType, documentId) as DbComment;
+    return await commentGetAuthorName(document) + ' left a new comment on the "' + startCase(await getCommentParentTitle(document)) + '" Subforum';
+  },
+  getIcon() {
+    return <CommentsIcon style={iconStyles}/>
+  },
```

Comment:
I've left in the code for in-app notifications in case we ever want them, but they are disabled by setting `allowedChannels: ["none", "email"]`